bin/pulseaudio-equalizer.in

```diff
@@ -30,12 +33,6 @@ SCRIPT_VERSION='4.0 (29.01.2017)'
 CONFIG_NAME="$CONFIG_DIR"/equalizerrc
 LOG_NAME="$HOME"/pulseaudio-equalizer.log
 
-# Don't execute the script with elevated privileges
-if [[ ! $EUID -ne 0 ]]; then
-  echo "This script must not be run as root!" 1>&2
-  exit 1
-fi
-
```

Comment:
Why? This would prevent the script to be run with sudo and breaking the setup. The only reason I see to justify this was if we wanted to use the equalizer in the root account which shouldn't happen as the root account really shouldn't be used for general purpose things.

Reply:
@FFY00 There seems to be some misunderstanding here. The root check isn't changed by this PR, only moved to the top of the script and simplified (avoiding a double negative) in https://github.com/pulseaudio-equalizer-ladspa/equalizer/pull/22/commits/e3a9e86a6ed957e6e4d8d6777488ce291ab0640a.

Reply:
:man_facepalming: I'm sorry, I looked at this in my phone and I missed that. Appart from what Gabriel said, this seems fine to me.